src/test/java/seedu/address/model/meeting/meeting/UniqueMeetingListTest.java

```diff
@@ -0,0 +1,172 @@
+package seedu.address.model.meeting.meeting;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.meeting.CommandTestUtil.VALID_DESCRIPTION_B;
+import static seedu.address.testutil.TypicalMeetings.MEETING_A;
+import static seedu.address.testutil.TypicalMeetings.MEETING_B;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.meeting.meeting.exceptions.DuplicateMeetingException;
+import seedu.address.model.meeting.meeting.exceptions.MeetingNotFoundException;
+import seedu.address.testutil.MeetingBuilder;
+
+
+class UniqueMeetingListTest {
+
+    private final UniqueMeetingList uniqueMeetingList = new UniqueMeetingList();
+
+    @Test
+    public void contains_nullPerson_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> uniqueMeetingList.contains(null));
+    }
+
+    @Test
+    public void contains_personNotInList_returnsFalse() {
+        assertFalse(uniqueMeetingList.contains(MEETING_A));
+    }
+
+    @Test
+    public void contains_personInList_returnsTrue() {
+        uniqueMeetingList.add(MEETING_A);
+        assertTrue(uniqueMeetingList.contains(MEETING_A));
+    }
+
+    //    @Test
+    //    public void contains_MeetingWithSameIdentityFieldsInList_returnsTrue() {
+    //        uniqueMeetingList.add(MEETING_A);
+    //        Meeting editedMeetingA = new MeetingBuilder(MEETING_A).withDescription(VALID_DESCRIPTION_B)
+    //                .build();
+    //        assertTrue(uniqueMeetingList.contains(editedMeetingA));
+    //    }
+
```

Comment:
Should we delete this comment or add a TODO comment?

Reply:
Maybe add a TODO comment

Reply:
Okayy

> Maybe add a TODO comment